examples/nanocoap_server/Makefile

```diff
@@ -47,6 +47,13 @@ endif
 HIGH_MEMORY_BOARDS := native same54-xpro mcb2388
 
 ifneq (,$(filter $(BOARD),$(HIGH_MEMORY_BOARDS)))
+  # enable separate response
+  USEMODULE += sock_aux_local
```

Comment:
Have you thought about wrapping this in a `nanocoap_sock_separate` module?

Reply:
Good idea!